packages/dd-trace/src/appsec/iast/path-line.js

```diff
@@ -7,7 +8,7 @@ const pathLine = {
 }
 
 const EXCLUDED_PATHS = [
-  '/node_modules/diagnostics_channel'
+  path.join(path.sep, 'node_modules', 'diagnostics_channel')
```

Comment:
Does the leading slash here translate into the filesystem root on a nix machine?

Reply:
This initial `/` is not for root. These `EXCLUDED_PATHS` are used to find the path in given filepath`filepath.indexOf(excludedPath)`. Is a way to exclude `project/node_modules/diagnostics_channel` without excluding `project/this_is_not_node_modules/diagnostics_channel`.

These exclusions weren't working properly in Windows.